### ISSUE TITLE
what happen ? unresolve IPv4Packet

### ISSUE THREAD

**Issue by Rgoogle** [libpnet/libpnet#684]
![image](https://github.com/libpnet/libpnet/assets/93381523/4b1f0215-c283-4db7-a7ce-60f9bd1e2f12)

I use the example in the respository   code .
and my enable all feature 

pnet = { version = "0.34.0", features = ["std", "pnet_transport",
    "ipnetwork", "appveyor",
    "default", "netmap",
    "pcap", "serde", "travis"] }

but not resolve yet .

**Comment by Paul-weqe** [comment]
Have you been able to fix this @Rgoogle  ?

**Comment by Paul-weqe** [comment]
Hmm...so first, Windows seems to be bringing issues to more than just you ([see here](https://github.com/libpnet/libpnet/issues/646)). The documentation will be improved on that.  I am on Ubuntu so I could have been of little help anyway. 

Now to the second problem. I will be using `MutableEthernetPacket` and `EthernetPacket` to illustrate. The problem comes out  because the structs such as `EthernetPacket` and `MutableEthernetPacket` and the rest of the packets are not manually coded, they are generated via the Packet Macro when the library is compiled, ([look here](https://github.com/libpnet/libpnet/blob/87f362dea40a0a58b2b1660c51038343c0731dea/pnet_macros/src/lib.rs#L57)). From line 57 it describes how they are automatically generated. 

Basically this means that when you look for their written code, such as by pressing `Ctrl` + `Enter`, the only place it can take you is to the Macro that produced it. If you really want to have a look at the struct that called the macro for e.g `MutableEthernetPacket` and `EthernetPacket` to be produced, most of them are inside [here](https://github.com/libpnet/libpnet/tree/87f362dea40a0a58b2b1660c51038343c0731dea/pnet_packet/src). 

I don't know if I have answered your question correctly. If not, let me know where I can elaborate.
